careers/ladder.md

```diff
@@ -28,73 +28,206 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
+
 ### IC1 Associate Engineer
 
+#### tl;dr: focuses on being an effective individual contributor
+
+#### Knowledge Leadership
+
 - Has broad knowledge of CS concepts.
-- Can complete well-defined and subdivided tasks.
 - Focuses on learning tools and processes.
+- Continues to develop individual contributor skills, including source control, editors, pairing and test-driven
+  development.
 - Routinely pushes code through the entire lifecycle from development to production.
-- Continues to develop individual contributor skills, including source control, editors or test-driven development.
+
+#### Impact
+
+- Can complete well-defined and subdivided tasks.
+- Tasks can be business or learning driven.
+
+#### Influence
+
+- Understands team mission, goals and backlog.
+- Builds relationship with team members.
+- Actively contributes to team rituals.
+
+#### Discretion
+
+- Guided by objectives of project or ongoing assignments.
+- Plans own work priorities on a regular basis with guidance.
 
 ### IC2 Engineer
 
+#### tl;dr: is an effective team member
+
+#### Knowledge Leadership
+
 - Consistently writes and delivers correct and clean quality code with guidance.
-- Self-sufficient and makes steady progress on tasks.
-- Knows when to ask for help and how to get unblocked.
-- Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
-- Owns a small-to-medium feature from technical design through completion.
+- Stewards a small-to-medium feature from technical design through completion.
+
+#### Impact
+
+- Makes steady, well-paced progress without the need for constant significant feedback from others.
+- Can reliably deliver tasks on the team's critical path.
+- Contributes small tasks to improve the engineering ecosystem.
+
+#### Influence
+
+- Effectively communicates in a distributed teams environment.
+- Presents facts, information, and the weight of the evidence to influence others.
+- Effectively partners with other engineers and with team's cross-functional partners (including PM, Designer, Data
+  Analyst).
 - Provides help and support outside of area under their responsibility.
 - Participates in the interviewing process of new team members.
 
+#### Discretion
+
+- Self-directed, plans own work priorities on a regular basis.
+- Knows when to ask for help and how to get unblocked.
+- Understands business impact of their work.
+
 ### IC3 Senior Engineer
 
-- Owns a service or large technology component.
-- Requires minimal direction or oversight, delivers quickly.
-- Understands elements of the business that their work supports.
+#### tl;dr: can drive change and impact at the team level
+
+#### Knowledge Leadership
+
 - Understands and makes well-reasoned design decisions and tradeoffs in their area.
 - Works in other areas of the codebase with guidance from owners of that system.
+- Takes responsibility for the maintenance and evolution of a significant service or technology.
```

Comment:
I see your point about individuals taking responsibility for systems, but the language here _does_ accurately describe how Artsy Engineering works. We have point-people listed in our project readme's for this reason – to explicitly identify who is responsible for the system. If we want to change that part of our engineering culture, I don't think that the career ladder is the best place to begin that change.

Reply:
Oh interesting! I thought that point people assignment was a relic of a prior organizational configuration - a legacy concept. This thinking is based on the fact that we organize systems underneath teams within the [Project List document within artsy/potential](https://github.com/artsy/potential/blob/master/Project-List.md), which is actively maintained.

I don't want this point to digress the PR much, but it would be good to clarify our position wrt here @joeyAghion and @SamRozen.

Reply:
Ah, interesting. I wasn't looking at it historically at all – for example, our support docs still refer to point people when describing how to handle incident response. 

I agree that there's some ambiguity between _teams_ handling a repo and _people_, though I will point out that many of the current point people have _been_ point people longer than our current product team structure has been in place (and, indeed, many will likely outlast our current team structure). I'm open to discussing these changes, though this might be best for a group discussion.

Reply:
I don’t think point people are a relic and are a very natural process. The point people that are indicated in project’s README’s are also actively maintained.

I’ve touched on team vs point people above already, so I’ll refer to that for my thoughts https://github.com/artsy/README/pull/188#discussion_r281603525

Reply:
Thanks for the reference – do you think we should attempt to have point people for a repository be _on the team_ that are the repository falls under on the project list?

I agree that there are some people that are more qualified than others to maintain a repo, but I also would like to see Artsy lean into having more autonomous teams that feel high degrees of ownership over the systems that they frequently develop against – I think aligning point people and teams would help achieve that.

Reply:
Unsure. I don’t see point people as owners in the sense of them making final calls, they are simply people that know much about a project and have expressed interest in being available to others when needed. I think it would be unfortunate if we’d have to remove people from that list if they do not want to be on a specific team.